### PR TITLE
Bcu 22#support vpc network

### PR DESCRIPTION
**Pull request by dactoankmapydev**
- Create,  update, delete VPC
- Datasource VPC: Input: VPC network name lấy vpc_network_id

### CODE REVIEW

File: bizflycloud/resource_bizflycloud_vpc_network_test.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package bizflycloud
```
Comment (greatbn) [inline bizflycloud/resource_bizflycloud_vpc_network_test.go:1]: Please add more test cases for this.

Comment (dactoankmapydev) [reply bizflycloud/resource_bizflycloud_vpc_network_test.go:1]: I have added more test cases for this.

File: examples/vpc_network/outputs.tf [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  value = bizflycloud_vpc_network.vpc_network.updated_at
+}
+
+output "mtu" {
+  value = bizflycloud_vpc_network.vpc_network.mtu
+}
+
+output "subnets" {
+  value = bizflycloud_vpc_network.vpc_network.subnets
+}
```
Comment (greatbn) [inline examples/vpc_network/outputs.tf:19]: no new line

File: examples/vpc_network/terraform.tfvars [hunk 3]
```diff
@@ -0,0 +1 @@
+vpc_network_name = "vpc_toannd"
```
Comment (greatbn) [inline examples/vpc_network/terraform.tfvars:1]: no new line

File: examples/vpc_network/variables.tf [hunk 4]
```diff
@@ -0,0 +1,3 @@
+variable "vpc_network_name" {
+  type = string
+}
```
Comment (greatbn) [inline examples/vpc_network/variables.tf:3]: no new line

File: examples/vpc_network/main.tf [hunk 5]
```diff
@@ -14,14 +14,12 @@ provider "bizflycloud" {
...
 data "bizflycloud_vpc_network" "vpc_network" {
-  id = bizflycloud_vpc_network.vpc_network.id
   name = bizflycloud_vpc_network.vpc_network.name
+}
+
+resource "bizflycloud_vpc_network" "vpc_network" {
+    name = var.vpc_network_name
+    description = "test vpc network"
+    is_default = false
 }
```
Comment (greatbn) [inline examples/vpc_network/main.tf:25]: no new line

**Review by greatbn — CHANGES_REQUESTED** [review]

**Pull request merged**